tests/console/suseconnect_scc.pm

```diff
@@ -35,11 +36,13 @@ sub run {
 
     # There are sporadic failures due to the command timing out, so we increase the timeout
     # and make use of retries to overcome a possible sporadic network issue.
-    script_retry("$cmd", retry => $retries, delay => 60, timeout => 180);
+    script_retry("$cmd", retry => $retries, delay => $delay, timeout => 180);
     # Check available extenstions (only present in sle)
-    assert_script_run q[SUSEConnect --list-extensions];
-    # What has been activated by default
-    assert_script_run q[SUSEConnect --list-extensions | grep -e '\(Activated\)'] if is_sle;
+    my $extensions = script_output_retry("SUSEConnect --list-extensions", retry => $retries, delay => $delay);
+    record_info('Extensions', $extensions);
+
+    # Make sure there is at least 1 module "Activated".
+    die("None of the modules are Activated") if ($extensions !~ m/Activated/ && is_sle);
```

Comment:
I just found out this is useles... the command outputs:
```
(Not available) The module/extension is not enabled on your RMT/SMT
(Activated)     The module/extension is activated on your system
```
so it will be always true, not sure why this check was there before....

Reply:
You would need to dig to the history. I am not sure, before there was a needle assertion there and as the tty ouput was not always completely seen on the screen it get you a nice overview what was activate.

Reply:
On the second thought there might be a regression introduce when SUSEConnect won't display the "activated" string there. Thus I would not personally remove it.

Reply:
ok, since this PR is about retry only, I will leave the logic it was there before.

Reply:
yes, it is useless due to the legend at the bottom shows what you mentioned, so always there is one hit.
we might need here something like `my $number = () = $extensions =~ /Activated/; die "None of the modules are Activated" if (is_sle() && $number <  2);` wdyt?

Reply:
Yeah, but looking at the VR https://openqa.suse.de/tests/8287612#step/suseconnect_scc/18,
there are no modules active, which should be also fine, shouldn't it?